ch02_Context.md

```diff
@@ -154,13 +154,13 @@ ontstaat.
 Beheer en ontwikkeling, in de brede zin, voor een standaard bevat ook
 onderwerpen als adoptie en certificering.
 
-**Beheer- en Ontwikkelmodel**
+**Beheer- en Ontwikkelmodel**    
```

Comment:
Waarom spaties na een titel?

Reply:
![image](https://github.com/user-attachments/assets/f2bd7ae7-f787-491a-a90e-3b8a9b0d03ef)

De formattering is verschillend bij deze kopjes, de spaties (aanvullen tot tab) zorgen ervoor dar de titels op aparte regel van de tekst staan. 
Als dit niet wenselijk is dan is dat ook goed. Zou wel adviseren om tot 1 format te komen.

Reply:
Dit zijn definities, niet kopjes (vandaar geen haakjes). 
Een WCAG-verantwoorde manier van definities aangeven is met `<dfn>`.
https://respec.org/docs/#definitions-and-linking